src/dlfind/dlf_dimer.f90

```diff
@@ -582,14 +582,16 @@ subroutine dlf_dimer_was_midpoint(trerun_energy,testconv)
   real(rk)              :: dlength,svar
   logical               :: tok,tconv
   real(rk),external     :: ddot
+  real(rk)              :: temp(1)
 ! **********************************************************************
   testconv=.true.
   ! ==================================================================
   ! This was the midpoint calculation, xcoords contains midpoint
   ! ==================================================================
 
   ! send information to set_tsmode
-  call dlf_formstep_set_tsmode(1,-1,glob%energy) ! send energy
+  temp(1) = glob%energy
```

Comment:
Nice one. The dlfind code has quite some adventurous interfaces. @kurt I am just noting this here because your idea of using libdlfind as available on Github would have fixed this (it was not available when we added the code - and we should have cleaned up the code ourselves). libdlfind has this fix
```
call dlf_formstep_set_tsmode(1,-1,(/glob%energy/)) ! send energy
```